ILRepack/ILRepack.csproj

```diff
@@ -10,9 +10,9 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>ILRepacking</RootNamespace>
     <AssemblyName>ILRepack</AssemblyName>
-    <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
```

Comment:
Cecil set to 4 ? sure ? maybe using the wrong configuration ?

As no feature of 4 is needed, compiling with 3.5 will allow ILRepack to be used of any platform w/o .Net4.
(or in a 3.5 project that needs it as a dep)

Not sure it's pretty common these days, we could also drop it, was just trying to understand the reason for that change.